scss/_patterns_side-navigation.scss

```diff
@@ -216,44 +247,89 @@
     }
   }
 
-  .p-side-navigation__link {
-    @include vf-focus;
+  .p-side-navigation__status {
+    margin-left: auto;
+    padding-left: $spv-inner--small;
+  }
 
-    // vf-highlight-bar is rendered above focus outline
-    // so we need to hide it on focus
-    &:focus::before {
-      display: none;
+  // Styles for markup in Discourse docs variant
+  .p-side-navigation--discourse {
```

Comment:
@anthonydillon Is it fine to name it as `p-side-navigation--discourse` (to explicitly state it's purpose) or do we want to be more generic with the name?

Reply:
Interesting, I would prefer to name it after its function and what it will be used on. Maybe `--extended`, `--raw`, `--cascade`?

Reply:
I agree that more functional name would be better, just couldn't find any that wouldn't be more enigmatic than helpful.
`--discourse` at least says where it's meant to be used, kind of enforces the feeling that this is an exception rather that the rule.
`--extended` `--raw` `--cascade` don't say much, it's hard to relate the name to what it may do. Maybe `--raw` is the closest (knowing the context that we allow styling raw HTML tags).

Maybe something around `--clean-html`, `--classless` `--classless-html`?

Reply:
That's what I was thinking. I think between `--raw` and `--clean-html` could work. It would be nice to be able to reuse this construct for other patterns and situations if needed.

Reply:
> That's what I was thinking. I think between `--raw` and `--clean-html` could work.

@anthonydillon `--raw-html` then? 😉